### ISSUE TITLE
core eq3btsmart integration

### ISSUE THREAD

**Issue by renarena** [dbuezas/eq3btsmart#113]
How is https://rc.home-assistant.io/integrations/eq3btsmart related to this integration?

**Comment by dbuezas** [comment]
It is based on this integration, but simplified (less entities and options)

**Comment by EuleMitKeule** [comment]
It's pretty much the same thing. I patched some of the code to conform to HA standards and increased the polling rate for status updates so ESP bluetooth proxies do not loose connection. I will try to get all of the missing stuff (schedule, away, boost, configuration, ...) merged in 2024.6. Currently only the climate entity is included.

**Comment by paulokow** [comment]
And battery status as separate entity pls.

**Comment by dbuezas** [comment]
It is a separate entity already

**Comment by paulokow** [comment]
@dbuezas yes, in this custom integration. The core eq3btsmart does not expose it separately AFAIK @EuleMitKeule

**Comment by EuleMitKeule** [comment]
Yes, not yet. But adding the remaining entities is planned. Also, issues regarding the core integration would be better placed in the core repository I would say.

**Comment by nilswieber** [comment]
@EuleMitKeule Is there any timeframe at which point these additional entities will be available in the core integration?

**Comment by EuleMitKeule** [comment]
@nilswieber I'll try to get some work on it done today!

**Comment by EuleMitKeule** [comment]
https://github.com/home-assistant/core/pull/130263

First PR that adds the binary sensor entities has been opened. Feel free to review it!

**Comment by EuleMitKeule** [comment]
Todo list for the core integration:

- [x] Climate entity
- [x] Away Switch
- [x] Boost Switch
- [x] Lock Switch
- [x] Window Binary Sensor
- [x] Battery Binary Sensor
- [x] Daylight Saving Time Binary Sensor
- [x] Away Hours Number
- [x] Away Temperature Number
- [x] Comfort Temperature Number
- [x] Eco Temperature Number
- [x] Offset Temperature Number
- [x] Window Open Temperature Number
- [x] Window Open Timeout Number
- [x] Away End Sensor Entity
- [x] Valve Sensor Entity
- [x] Coordinator for data fetching
- [ ] Set Away Until Service

Maybe:
- Schedule Sensor Entity
- Set Schedule Service

Probably won't add:
- Connection Switch
- Fetch Schedule Button
- RSSI Sensor Entity (Will probably integrate this into the state polling)

Won't add:
- Connected Sensor (Reflects whether the other entities are available)
- Busy Sensor (Not needed with new library)
- Fetch Button (Won't be needed when new coordinator is finished)
- Lock Entity (Changed to Switch Entity)
- MAC Sensor (Can be accessed via device properties)
- Serial Number Sensor (Can be accessed via device properties)
- Firmware Version Sensor (Can be accessed via device properties)
- Options to configure the current/target temperature, since this is unwanted by HA
- Current temperature (it only shows differences between the assumed and actual state due to connection delay which is not the use case for this attribute)
- Options flow (all config options are obsolete)
